examples/run/cpu/seq_example.cpp

```diff
@@ -38,8 +43,31 @@ int seq_run(const traccc::full_tracking_input_config& i_cfg,
             const traccc::common_options& common_opts,
             const traccc::detector_input_options& det_opts) {
 
-    // Read the surface transforms
-    auto surface_transforms = traccc::io::read_geometry(det_opts.detector_file);
+    // Memory resource used by the application.
+    vecmem::host_memory_resource host_mr;
+
+    // Construct a detector object.
+    detray::io::detector_reader_config reader_cfg{};
+    reader_cfg.add_file(traccc::io::data_directory() + det_opts.detector_file);
+    if (!det_opts.material_file.empty()) {
+        reader_cfg.add_file(traccc::io::data_directory() +
+                            det_opts.material_file);
+    }
+    if (!det_opts.grid_file.empty()) {
+        reader_cfg.add_file(traccc::io::data_directory() + det_opts.grid_file);
+    }
+    auto [detector, _] =
+        detray::io::read_detector<detray::detector<> >(host_mr, reader_cfg);
+
+    // Construct an "old style geometry" from the detector object.
+    traccc::geometry surface_transforms =
+        traccc::io::alt_read_geometry(detector);
+
+    // Construct a map from Acts surface identifiers to Detray barcodes.
+    std::map<uint64_t, detray::geometry::barcode> barcode_map;
```

Comment:
Should it not better be
```suggestion
    std::map<std::uint64_t, detray::geometry::barcode> barcode_map;
```
like for ```std::size_t```?

Reply:
I was wondering about that myself as well. 🤔 In the rest of the I/O code we use `uint64_t` at the moment, that's why I used the same here as well. But yes, I fully agree, that all of these places should be migrated to `std::uint64_t` instead.

Reply:
A detail for later then :)